src/functionalTest/resources/features/F-205 - Validate NoC Questions/S-205.2.td.json

```diff
@@ -0,0 +1,33 @@
+{
+  "_guid_": "S-205.2",
+  "_extends_": "S-205_Test_Data_Base",
+  "specs" : [
+    "Richard - with the ability to create a case for a particular jurisdiction within an organisation",
+    "Matt - with a pui-CAA within a different organisation from Richard",
+    "the configuration for the case type of Case C1 is such that there is a NoCRequest event available to users with the IDAM ID Caseworker-CAA when the case is in the state established following case creation",
+    "is made by Matt to validate the answers provided to the NoC questions in order to become the representative for Mario on C1",
+    "contains all correct answers"
+  ],
+
+  "users": {
+    "invokingUser": {
+      "_extends_": "ACA_Users_CAA_Matt"
+    }
+  },
```

Comment:
If two users in specs list should have two users here (with order matching the order in feature file).